src/languages/N1qlFormatter.js

```diff
@@ -19,9 +19,10 @@ const reservedWords = [
 ];
 
 const reservedToplevelWords = [
-    "SELECT", "FROM", "WHERE", "SET", "ORDER BY", "GROUP BY", "LIMIT", "VALUES", "EXPLAIN UPDATE", "UPDATE",
-    "HAVING", "EXPLAIN DELETE FROM", "DELETE FROM", "UNION ALL", "UNION", "EXCEPT ALL", "EXCEPT", "INTERSECT ALL", "INTERSECT",
-    "INFER", "EXPLAIN UPSERT", "UPSERT", "MERGE", "PREPARE", "LET", "USE KEYS", "UNNEST", "NEST", "INSERT INTO"
+    "SET SCHEMA", "SET CURRENT SCHEMA", "SELECT", "FROM", "WHERE", "SET", "ORDER BY", "GROUP BY", "LIMIT", "VALUES",
```

Comment:
Would be better to have all the "SET" things together.

Thinking further... would be good to actually order them all alphabetically.